client/verta/verta/_cli/registry/get.py

```diff
@@ -56,4 +56,25 @@ def get_model(model_name, output, workspace):
 def get_model_version(model_name, version_name, output, workspace):
     """Get detailed information about a model version.
     """
-    pass
+    client = Client()
+
+    try:
+        client.get_registered_model(model_name, workspace=workspace)
+    except ValueError:
+        raise click.BadParameter("model {} not found".format(model_name))
+
+    try:
+        version = client.get_registered_model_version(name=version_name)
```

Comment:
Please change this to use the `RegisteredModel` from line 62, thanks!

Reply:
Fixed.

Reply:
Forgot to push the commit =)) _Now_ it's fixed.